src/data_retrieval/__main__.py

```diff
@@ -57,52 +58,87 @@ def get_cds_era5():
     file_name: str = (
         "era5_total_precipitation_global_1981_2023_all_months.grib"
     )
-    output_path: str = os.path.expanduser("~/Downloads/era5_global_data")
-    setup_output_path(output_path)
+    blob_path = "/raw/glb/era5/era5-total-precipitation-1981-2023.grib"
```

Comment:
I do not think it is a good idea to kill already existing feature. Some open-source users may not have the assess to the blob store.

Please remove the hard coded path add add `argparse` CLI argument to switch between local and cloud download.